update.sh

```diff
@@ -34,6 +44,77 @@ packages_found=
 packages_debs=
 packages_sha256=
 
+check_key() {
+    # param 1 = keyfile
+    # param 2 = key fingerprint
+    KEY_FILE="$1"
+    KEY_FINGERPRINT="$2"
+
+    if [ ! -f ${KEY_FILE} ] || [ "${KEY_FINGERPRINT}" = "" ] ; then
+        echo "One or more required parameters for check_key() is missing"
```

Comment:
this error can be misleading, because the existence of KEY_FILE is tested, not whether the parameter is missing